<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="displayLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionWire_Frame"/>
    <addaction name="actionShade"/>
    <addaction name="actionShade_Curvature"/>
    <addaction name="actionShade_Developable"/>
    <addaction name="actionShade_Zebra"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Control_Net"/>
    <addaction name="actionShow_Interior_Edges"/>
    <addaction name="actionShow_Control_Curves"/>
    <addaction name="actionShow_Curvature"/>
    <addaction name="actionShow_Normals"/>
    <addaction name="actionDraw_Mirror"/>
    <addaction name="actionShade_Underwater"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionWire_Frame">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wire Frame</string>
   </property>
  </action>
  <action name="actionShade">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shade</string>
   </property>
  </action>
  <action name="actionShade_Curvature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Shade Curvature</string>
   </property>
  </action>
  <action name="actionShade_Developable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Shade Developable</string>
   </property>
  </action>
  <action name="actionShade_Zebra">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Shade Zebra</string>
   </property>
  </action>
  <action name="actionShow_Control_Net">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Control Net</string>
   </property>
  </action>
  <action name="actionShow_Interior_Edges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Interior Edges</string>
   </property>
  </action>
  <action name="actionShow_Control_Curves">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Control Curves</string>
   </property>
  </action>
  <action name="actionShow_Curvature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Curvature</string>
   </property>
  </action>
  <action name="actionShow_Normals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Normals</string>
   </property>
  </action>
  <action name="actionDraw_Mirror">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw Mirror</string>
   </property>
  </action>
  <action name="actionShade_Underwater">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shade Underwater</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
